src/components/icon/index.tsx

```diff
@@ -50,16 +55,36 @@ const Icon = (props: Props) => {
     return source;
   }, [source, assetGroup, assetName]);
 
+  if (typeof source === 'string' && isBase64ImageContent(source) && Constants.isWeb) {
+    return (
+      <Image
+        {...others}
+        ref={ref}
+        source={iconSource}
+        style={[
+          style,
+          margins,
+          iconSize,
+          shouldFlipRTL && styles.rtlFlipped,
+          !!tintColor && {
+            tintColor
+          }
+        ]}
+      />
+    );
+  }
```

Comment:
It looks like this render is exactly the same as the default render at the bottom..

Reply:
I'll move it into `renderImage` function.
Should I move the Svg into `renderSvg` function also?

Reply:
Sure, it can make a little more organized 👌